bedrock/mozorg/templates/mozorg/cms/about/leadership.html

```diff
@@ -18,40 +26,46 @@
 {% endblock %}
 
 {% block content %}
-  <div id="main-content" class="mzp-l-content mzp-has-sidebar mzp-l-sidebar-left">
-    <aside class="mzp-l-sidebar">
-      {% set nav_bar = [
-        (fullpageurl(page), 'executive-team', page.title),
-      ] -%}
-      {% for section in page.leadership_sections %}
-        {{ nav_bar.append(('#' ~ section.value.id, section.value.id, section.value.title)) }}
-      {% endfor %}
-
-      {{ sidemenu_lists([nav_bar]) }}
-    </aside>
-
-    <main class="mzp-l-main">
-      {% for section in page.leadership_sections %}
-        <section class="leadership-section" id="{{ section.value.id }}">
-          {% set first_group = section.value.leadership_group[0] %}
-          <h2 class="leadership-title{% if first_group.title %} compact{% endif %}">{{ section.value.title }}</h2>
-
-          {% for group in section.value.leadership_group %}
-            <section>
-              {% if group.title %}
-                <h2 class="group-title">{{ group.title }}</h2>
-              {% endif %}
-              <div class="gallery mgmt-corp">
-                {% for leader in group.leaders %}
-                  {% include_block leader %}
-                {% endfor %}
-              </div>
-            </section>
-          {% endfor %}
-        </section>
-      {% endfor %}
-    </main>
-  </div>
-
-  <div class="mzp-u-modal-content"></div>
+  <main class="mzp-l-main">
+    <div id="main-content" class="mzp-l-content mzp-has-sidebar mzp-l-sidebar-left">
+      <aside class="mzp-l-sidebar">
+        {% set nav_bar = [
+          (fullpageurl(page), 'executive-team', page.title),
+        ] -%}
+        {% for section in page.leadership_sections %}
+          {{ nav_bar.append(('#' ~ section.value.id, section.value.id, section.value.title)) }}
+        {% endfor %}
+
+        {{ sidemenu_lists([nav_bar]) }}
+      </aside>
+
+      <main class="mzp-l-main">
```

Comment:
Does it matter that there are two `main` nodes here? one here and one at the top of the diff?

Reply:
Oh good call, you're right there should only be one. I'll update!

Reply:
@stevejalim updated, thanks